ui/src/App.js

```diff
@@ -30,6 +30,7 @@ import config from './config'
 import { setDispatch, startEventStream } from './eventStream'
 import { HotKeys } from 'react-hotkeys'
 import { keyMap } from './hotkeys'
+import ChangeColor from './common/PWA'
```

Comment:
I think this component should leave in the root (`ui/src`) as it is only used by the `App.js` file, not really a "common" component. Also, I'd rename `PWA.js` to `ChangeColor` as it is the only component exported by the file